src/components/tools/tool-tile.tsx

```diff
@@ -283,16 +283,22 @@ export class ToolTileComponent extends BaseComponent<IProps, {}> {
       }
     };
 
+    // support dragging a tile without selecting it first
+    const dragTileIds = selectedTileIds.slice();
+    if (dragTileIds.indexOf(model.id) < 0) {
+      dragTileIds.push(model.id);
+    }
+
```

Comment:
Nice solution.  I was thinking we were going to have to expose the curriculum as a document somehow.